Unity/Assets/Scripts/GameManager.cs

```diff
@@ -0,0 +1,414 @@
+    [SerializeField] Text _battleText;
+
+    int _playerCreatureNumber;
+    int _opponentCreatureNumber;
+    int _playerScaledPower;
+    int _opponentScaledPower;
+
+    enum Field {   
+        Tokyo = 0,
+        Savannah = 1,
+        PacificOcean = 2,
+        Space = 3,
+    }
+    enum Character
+    {
+        Elephant = 0,
+        Lion = 1,
+        Zebra = 2,
+        Dolphin = 3,
+        Orca = 4,
+        Human = 5,
+    }
+
+    class Creature{
+        public string name;
+        public int power;
+        public string field;
+
+        public Creature(string name, int power, string field)
+        {
+            this.name = name;
+            this.power = power;
+            this.field = field;
+        }
+    }
+
+    void Start()
+    {
+        _characterButtons[(int)Character.Elephant].onClick.AddListener(()=> SetImage((int)Character.Elephant));
+        _characterButtons[(int)Character.Lion].onClick.AddListener(()=> SetImage((int)Character.Lion));
+        _characterButtons[(int)Character.Zebra].onClick.AddListener(()=> SetImage((int)Character.Zebra));
+        _characterButtons[(int)Character.Dolphin].onClick.AddListener(()=> SetImage((int)Character.Dolphin));
+        _characterButtons[(int)Character.Orca].onClick.AddListener(()=> SetImage((int)Character.Orca));
+        _characterButtons[(int)Character.Human].onClick.AddListener(()=> SetImage((int)Character.Human));
+
+        _playerDeleteButton.onClick.AddListener(()=> SetPlayerDelete());
+        _opponentDeleteButton.onClick.AddListener(()=> SetOpponentDelete());
+
+        _nextFieldImageButton.onClick.AddListener(()=> ChangeNextFieldImage());
+        _returnFieldImageButton.onClick.AddListener(()=> ChangeReturnFieldImage());
+
+        _startGameButton.onClick.AddListener(()=> StartGame());
+        _nextGameButton.onClick.AddListener(()=> NextGame());
+        _finishGameButton.onClick.AddListener(()=> FinishGame());
+
+        _battleGameObject.SetActive(false);
+
+        _playerDeleteButton.gameObject.SetActive(false);;
+        _opponentDeleteButton.gameObject.SetActive(false);
+        _nextGameButton.gameObject.SetActive(false);
+        
+        _playerCreatureImage.enabled = false;
+        _opponentCreatureImage.enabled = false;
+
+        _battleText.enabled = false;
+        _battleTextImage.enabled = false;
+    }
+
+    void SetImage(int number)
+    {
+        switch (number)
+        {
+            case 0:
+                CharacterDisplay(number);
+                break;
+            case 1:
+                CharacterDisplay(number);
+                break;
+            case 2:
+                CharacterDisplay(number);
+                break;
+            case 3:
+                CharacterDisplay(number);
+                break;
+            case 4:
+                CharacterDisplay(number);
+                break;
+            case 5:
+                CharacterDisplay(number);
+                break;
+        }
+    }
```

Comment:
以下と同じなので、switchしている意味がないように見えますね...
```suggestion
    void SetImage(int number)
    {
        CharacterDisplay(number);       
    }
```